<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateDialog</class>
 <widget class="QDialog" name="TemplateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шаблон планировщика</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="0" colspan="8">
    <widget class="COrgSrtuctureJobsView" name="tblJobs"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QComboBox" name="cmbTimelinePeriod">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Один план</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Нечет/чет</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Произвольный график</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Одна неделя</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Две недели</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Три недели</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Четыре недели</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;По</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>В период &amp;с</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="8">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="8">
    <widget class="QCheckBox" name="chkRemoveExistingJobs">
     <property name="text">
      <string>Удалить предыдущее расписание</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="edtTimelineCustomLength">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="lblTimelineCustomLength">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>длительность</string>
     </property>
     <property name="buddy">
      <cstring>edtTimelineCustomLength</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>449</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="6">
    <widget class="QCheckBox" name="chkFillRedDays">
     <property name="text">
      <string>Заполнять выходные и праздничные дни</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>COrgSrtuctureJobsView</class>
   <extends>QTableView</extends>
   <header location="global">Resources/OrgStructureJobs.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbTimelinePeriod</tabstop>
  <tabstop>edtTimelineCustomLength</tabstop>
  <tabstop>chkFillRedDays</tabstop>
  <tabstop>tblJobs</tabstop>
  <tabstop>chkRemoveExistingJobs</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TemplateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>676</x>
     <y>702</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TemplateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>702</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
